### PR TITLE
updated uowMock declaration

### LINKED ISSUE #22: Deployment Fails OpportunitiesTest
When deploying the sample code to an org using the deploy tool, there is the following deployment error: 

deploy/classes/OpportunitiesTest.cls(38,38):Invalid type: fflib_SObjectMocks.SObjectUnitOfWork

I believe this is because the fflib_SObjectMocks pattern has been replaced, and line 38 on the test class should be:

fflib_SObjectUnitOfWork uowMock = 
    (fflib_SObjectUnitOfWork) mocks.mock(fflib_SObjectUnitOfWork.class);

### PR DESCRIPTION
**Pull request by ctchipps**
Replaced the uowMock to follow (what I think) is supposed to be the correct pattern

Fixes #22

### CODE REVIEW

**Comment by ctchipps** [comment]
@ImJohnMDaniel Hey John, do you have any time to review this PR? Thank you so much!

File: sfdx-source/apex-common-samplecode/test/classes/domains/OpportunitiesTest.cls [hunk 1]
```diff
@@ -35,8 +35,7 @@ private class OpportunitiesTest
 	{
 		// Create mocks
 		fflib_ApexMocks mocks = new fflib_ApexMocks();
-		fflib_ISObjectUnitOfWork uowMock = new fflib_SObjectMocks.SObjectUnitOfWork(mocks);
-
+		fflib_SObjectUnitOfWork uowMock = (fflib_SObjectUnitOfWork) mocks.mock(fflib_SObjectUnitOfWork.class);
```
Comment (ImJohnMDaniel) [inline sfdx-source/apex-common-samplecode/test/classes/domains/OpportunitiesTest.cls:38]: @ctchipps -- please use the UOW interface (`fflib_ISObjectUnitOfWork`) and not the implementation itself when mocking.

Comment (ctchipps) [reply sfdx-source/apex-common-samplecode/test/classes/domains/OpportunitiesTest.cls]: Updated to use the interface

**Comment by ctchipps** [comment]
Thank you all!

**Pull request merged**